pkg/lakom/verifysignature/admission_test.go

```diff
@@ -29,58 +28,29 @@ import (
 )
 
 var (
-	deploymentGVK   = metav1.GroupVersionKind{Group: "apps", Kind: "Deployment", Version: "v1"}
-	podGVK          = metav1.GroupVersionKind{Group: "", Kind: "Pod", Version: "v1"}
-	imageDigest     = "gcr.io/projectsigstore/cosign@sha256:f9fd5a287a67f4b955d08062a966df10f9a600b6b8583fd367bce3f1f000a429"
-	cosignPublicKey = `-----BEGIN PUBLIC KEY-----
-MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEhyQCx0E9wQWSFI9ULGwy3BuRklnt
-IqozONbbdbqz11hlRJy9c7SG+hdcFl9jE9uE/dwtuwU2MqU9T/cN0YkWww==
------END PUBLIC KEY-----
-`
-
-	cosignConfig = lakomconfig.Config{
-		PublicKeys: []lakomconfig.Key{
-			{
-				Name:      "test",
-				Key:       cosignPublicKey,
-				Algorithm: lakomconfig.RSAPKCS1v15SHA256,
-			},
-		},
-	}
+	deploymentGVK = metav1.GroupVersionKind{Group: "apps", Kind: "Deployment", Version: "v1"}
+	podGVK        = metav1.GroupVersionKind{Group: "", Kind: "Pod", Version: "v1"}
 
-	scheme    *runtime.Scheme
 	ctrl      *gomock.Controller
 	mgr       *mockmanager.MockManager
 	apiReader *mockclient.MockReader
 )
 
-var _ = BeforeSuite(func() {
-	scheme = runtime.NewScheme()
-	err := corev1.AddToScheme(scheme)
-	Expect(err).ToNot(HaveOccurred())
-
-	dirPath, err := os.MkdirTemp("", "verifysignature_test")
-	Expect(err).ToNot(HaveOccurred())
-	DeferCleanup(func() {
-		err := os.RemoveAll(dirPath)
-		Expect(err).ToNot(HaveOccurred())
-	})
-})
-
 var _ = Describe("Admission Handler", func() {
 	var (
-		ctx     = context.Background()
-		logger  logr.Logger
-		handler admission.Handler
-		pod     = &corev1.Pod{
+		ctx          = context.Background()
+		logger       logr.Logger
+		handler      admission.Handler
+		cosignConfig lakomconfig.Config
+		pod          = &corev1.Pod{
```

Comment:
```suggestion
		pod          *corev1.Pod
```

As the variable is initialized in `BeforeEach`, you should just declare it here.

Reply:
I've applied the same fix as the previous occurrence of this issue